gulp-tasks/wfContributors.js

```diff
@@ -59,6 +64,30 @@ function buildIndex(contributors) {
   fs.writeFileSync(DEST_LIST, result);
 }
 
+function buildIndividualPages(contributors) {
+  gutil.log(' ', 'Building individual pages of all contributors...');
+  var files = wfHelper.getFileList(GLOBAL.WF.src.content, ['**/*.md']);
+  var filesByAuthor = wfHelper.splitByAuthor(files);
+
+  var keys = Object.keys(contributors);
+  keys.forEach(function(key) {
+    if (!(key in filesByAuthor)) {
+      return;
+    }
+    filesByAuthor[key].sort(wfHelper.updatedComparator);
+    var name = contributors[key].name;
+    var ts = fs.readFileSync(TEMPLATE_ARTICLE_LIST, 'utf8');
+    var template = Handlebars.compile(ts);
+    var context = {
+      title: 'Latest contributions from ' + name.given + ' ' + name.family,
```

Comment:
if `name.family` is empty, you get `undefined` - Surma is the perfect example, need to handle single named people. :(

Reply:
Done